tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java

```diff
@@ -94,6 +94,10 @@ public TiTableInfo(
     primaryKeyColumn = primaryKey;
   }
 
+  public boolean isNotView() {
+    return this.viewInfo == null;
```

Comment:
better use `!isView()` so that we don't need to modify multiple lines in future.